### PR TITLE
Urban: show first 3 answers + 1 random answer from rest of them

### PR DESCRIPTION
**Pull request by wdv4758h**

### CODE REVIEW

**Comment by wdv4758h** [comment]
looks like iTaigiDict testing timeout from time to time

**Comment by M157q** [comment]
Why random?
I don't see any meaning doing this as a dictionary.
Why not just show the first result and show all results in verbose mode?

File: zdict/dictionaries/urban.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import json
+from random import randint
```
Comment (iblis17) [inline zdict/dictionaries/urban.py]:
```suggestion

from random import randint
```

**Comment by wdv4758h** [comment]
I'm closing this one for now, since it's outdated. We can deal with issue #342 in a new PR if it's still needed.

**Pull request closed without merging**